src/dotnet/HQ.Server/Program.cs

```diff
@@ -264,6 +264,13 @@
     Cron.Daily(8),
     recurringJobOptions);
 
+
+recurringJobManager.AddOrUpdate<PlanServiceV1>(
+    nameof(PlanServiceV1.BackgroundSendPlanSubmissionReminderEmail),
+    (t) => t.BackgroundSendPlanSubmissionReminderEmail(Period.Today, CancellationToken.None),
+    Cron.Daily(10),
```

Comment:
We don't want it to run every day at 10, only weekdays. If there isn't a helper function, you can use this CRON string to schedule weekdays: https://crontab.guru/every-weekday